src/Command/MarkMigrated.php

```diff
@@ -60,7 +60,9 @@ protected function configure()
             ));
         $this->addOption('plugin', 'p', InputArgument::OPTIONAL, 'The plugin the file should be created for')
             ->addOption('connection', 'c', InputArgument::OPTIONAL, 'The datasource connection to use')
-            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in');
+            ->addOption('source', 's', InputArgument::OPTIONAL, 'The folder where migrations are in')
+            ->addOption('skip-to', 'u', InputArgument::OPTIONAL, 'If present it will mark all versions until the given one')
+            ->addOption('up-to', 't', InputArgument::OPTIONAL, 'If present it will mark all versions including the given one');
```

Comment:
Two things : 
- As @dakota hinted in his comment (https://github.com/cakephp/migrations/issues/129#issuecomment-144730650), we should use ``--target`` as the option name to make it consistent with phinx implementation of the ``migrate`` and ``rollback`` commands. And to give the option of excluding the version passed, we could add another flag ``--exclude`` that would only work with the ``--target`` one to have something like this :

```
# Mark all migrations up to 123456789 as migrated
bin/cake migrations mark_migrated 123456789 --target

# Mark all migrations up to the one before 123456789
bin/cake migrations mark_migrated 123456789 --target --exclude
```
Just using the ``--exclude`` one (without the ``--target`` one should do nothing (or output an error stating that this flag should be used with the other one)

(I'm open to suggestions about this).

- Since this option is used as a boolean flag, the third param should be ``InputOption::VALUE_NONE``